src/mtcp/mtcp_restart.c

```diff
@@ -417,7 +417,7 @@ static void restart_fast_path()
   asm volatile(
    // 96 = offsetof(RestoreInfo, rinfo.restorememoryareas_fptr)
```

Comment:
Please also update the comment:  `// 104 - offsetof(...)`.
Thanks for catching this bug fix of mine.  But I thought I had tested my code and it worked.  Does the restart really fail without this?  If so, thanks again.